packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2471,9 +2539,17 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
   @override
   final bool allowNameWrap;
 
+  bool _isExpandingPropertyValuesForJson(DiagnosticsSerialisationDelegate delegate) {
```

Comment:
maybe it violates the style guide but
expandPropertyValuesForJson seems like a simpler name for this.